countries-geojson/countries_geojson.py

```diff
@@ -0,0 +1,35 @@
+import csv, json
+from re import M
+from geojson import Feature, FeatureCollection, Point
+
+csvFilePath = "/Users/cpapalaz/Desktop/Mangrove_Countries.csv"
+jsonFilePath = "/Users/cpapalaz/Desktop/Mangrove_Countries.json"
```

Comment:
[Best practice](https://peps.python.org/pep-0008/#function-and-variable-names) in python would be to name variables with snake case (words lowercase and separated by underscores).

Hard coding paths which are specific to your environment isn't ideal. Can you make these relative paths? I know relative paths worked in my example which I wrote in a Jupyter notebook but it's not always straightforward in python. If the suggestion below doesn't work, maybe try something like [this](https://stackoverflow.com/a/918178/3704248).

```suggestion
csv_file_path = "Mangrove_Countries.csv"
json_file_path = "../mrtt-ui/src/data/mangrove_countries.json"
```